client/src/linkManager.js

```diff
@@ -297,11 +297,13 @@ export class linkManager extends LuigiClientBase {
       }.bind(this)
     );
 
+    const hasIntent = path.toLowerCase().includes('#?intent=');
     const pathExistsMsg = {
       msg: 'luigi.navigation.pathExists',
       data: Object.assign(this.options, {
         id: currentId,
         link: path,
+        intent: hasIntent,
```

Comment:
Line [300](https://github.com/SAP/luigi/pull/2267/files#diff-ee577ceabc2718709233de3e6b792c3684f0f7cd122e76a9a80471b2eedb188fR300) and line [76](https://github.com/SAP/luigi/blob/06c58b0083e2f0b9cb3b345f67ebe869e262c08e/client/src/linkManager.js#L76) are doing the same.  
Could we move it out and reuse it? (Maybe `helpers`?)
So we can inline function as suggestion. Would this be better? 
```suggestion
        intent: hasIntent(path),
```

Reply:
I added the helper function.

Reply:
Well done. Thanks